modules/components/src/AdvancedSqonBuilder/filterComponents/TermFilter.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import Component from 'react-component-component';
+import { sortBy } from 'lodash';
+import './FilterContainerStyle.css';
+import {
+  getOperationAtPath,
+  setSqonAtPath,
+  FIELD_OP_DISPLAY_NAME,
+  TERM_OPS,
+} from '../utils';
+import { TermAgg } from '../../Aggs';
+import TextFilter from '../../TextFilter';
+import { inCurrentSQON } from '../../SQONView/utils';
+import { FilterContainer } from './common';
+
+const AggsWrapper = ({ children }) => (
+  <div className="aggregation-card">{children}</div>
+);
+
+export const TermFilterUI = ({
```

Comment:
it's this guy: 
![screen shot 2019-01-31 at 6 19 53 pm](https://user-images.githubusercontent.com/11821106/52092209-d3c00b80-2584-11e9-8ae2-834f5edc8acb.png)